Python 3.10.8 (tags/v3.10.8:aaaf517, Oct 11 2022, 16:50:30) [MSC v.1933 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license()" for more information.

==================== RESTART: C:\Users\HP\Desktop\112233.py ====================
hello
i am here
hello

=================== RESTART: C:\Users\HP\Desktop\112233.py ==================
hi how are u??
hello
i am here
hello

=================== RESTART: C:\Users\HP\Desktop\112233.py ==================
hi how are u??
hello
hello

=================== RESTART: C:\Users\HP\Desktop\112233.py ==================
hi how are u??
hello

=================== RESTART: C:\Users\HP\Desktop\112233.py ==================
hi how are u??
hello
i am here
hello

=================== RESTART: C:\Users\HP\Desktop\112233.py ==================
hi how are u??
hello
i am here
hello

=================== RESTART: C:\Users\HP\Desktop\112233.py ==================
hi how are u??
hello
i am here
hello

=================== RESTART: C:\Users\HP\Desktop\112233.py ==================
hi how are u??
hello
i am here
hello

=================== RESTART: C:\Users\HP\Desktop\112233.py ==================
hi how are u??
hello
i am here
hello

=================== RESTART: C:\Users\HP\Desktop\112233.py ==================
hi how are u??
hello
i am here
hello
Traceback (most recent call last):
  File "C:\Users\HP\Desktop\112233.py", line 7, in <module>
    pint(hello.__doc__)
NameError: name 'pint' is not defined. Did you mean: 'print'?

=================== RESTART: C:\Users\HP\Desktop\112233.py ==================
hi how are u??
hello
i am here
hello
None

=================== RESTART: C:\Users\HP\Desktop\112233.py ==================
hi how are u??
hello
i am here
hello
defing feature

===================== RESTART: C:/Users/HP/Desktop/46.py ====================

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
total
Traceback (most recent call last):
  File "C:/Users/HP/Desktop/46.py", line 4, in <module>
    add(10,20)
TypeError: add() takes 0 positional arguments but 2 were given

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
Traceback (most recent call last):
  File "C:/Users/HP/Desktop/46.py", line 3, in <module>
    add(10,20)
TypeError: add() takes 0 positional arguments but 2 were given

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
total
Traceback (most recent call last):
  File "C:/Users/HP/Desktop/46.py", line 6, in <module>
    add(10,20)
TypeError: add() takes 0 positional arguments but 2 were given

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
total
total

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
total 30
total 5

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
total 30
total 50

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
total 30
total 50

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
total 5
total 24
total 5

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
total= 5
total= 24
total= 5

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
total= 5
total= 24
total= 5

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
total= 5
total= 24
total= 10

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
total= 5
total= 24
total= 10

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
total= 15
total= 34
total= 20

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
total= 15
total= 34
total= 44

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
total= 6
total= 8
total= 11

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
total= 8
total= 10
total= 13

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
total= 6
total= 8
total= 11

===================== RESTART: C:/Users/HP/Desktop/46.py ====================

===================== RESTART: C:/Users/HP/Desktop/46.py ====================

===================== RESTART: C:/Users/HP/Desktop/46.py ====================

===================== RESTART: C:/Users/HP/Desktop/46.py ====================

===================== RESTART: C:/Users/HP/Desktop/46.py ====================

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
94

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
Traceback (most recent call last):
  File "C:/Users/HP/Desktop/46.py", line 7, in <module>
    print(add())
TypeError: add() missing 2 required positional arguments: 'a' and 'b'

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
Traceback (most recent call last):
  File "C:/Users/HP/Desktop/46.py", line 7, in <module>
    print(add(1))
TypeError: add() missing 1 required positional argument: 'b'

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
5

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
5

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
12

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
Traceback (most recent call last):
  File "C:/Users/HP/Desktop/46.py", line 8, in <module>
    if a>=0:
NameError: name 'a' is not defined

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
Traceback (most recent call last):
  File "C:/Users/HP/Desktop/46.py", line 8, in <module>
    if a>=0:
NameError: name 'a' is not defined

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
wrong value

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
wrong value

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
wrong value
Traceback (most recent call last):
  File "C:/Users/HP/Desktop/46.py", line 12, in <module>
    ass(4,6)
NameError: name 'ass' is not defined. Did you mean: 'abs'?

\===================== RESTART: C:/Users/HP/Desktop/46.py ====================
wrong value
(4, 6)

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
(4, 6)

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
(4, 6)
<class 'tuple'>

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
(4, 6)
<class 'tuple'>
(88, 66)
<class 'tuple'>
x=(1,2,3,4,5)
type(x)
<class 'tuple'>
dir(x)
['__add__', '__class__', '__class_getitem__', '__contains__', '__delattr__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__getitem__', '__getnewargs__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__iter__', '__le__', '__len__', '__lt__', '__mul__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__rmul__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', 'count', 'index']
class(x)
SyntaxError: invalid syntax
a
Traceback (most recent call last):
  File "<pyshell#4>", line 1, in <module>
    a
NameError: name 'a' is not defined
x
(1, 2, 3, 4, 5)
x.count()
Traceback (most recent call last):
  File "<pyshell#6>", line 1, in <module>
    x.count()
TypeError: tuple.count() takes exactly one argument (0 given)
x.count("")
0
x.count("2")
0

new=list(a)
Traceback (most recent call last):
  File "<pyshell#10>", line 1, in <module>
    new=list(a)
NameError: name 'a' is not defined
print (type(a))
Traceback (most recent call last):
  File "<pyshell#11>", line 1, in <module>
    print (type(a))
NameError: name 'a' is not defined
nnew=list(new)
Traceback (most recent call last):
  File "<pyshell#12>", line 1, in <module>
    nnew=list(new)
NameError: name 'new' is not defined
new=list(x)
print (type(x))
<class 'tuple'>
print(type(new))
<class 'list'>
dir(new)
['__add__', '__class__', '__class_getitem__', '__contains__', '__delattr__', '__delitem__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__getitem__', '__gt__', '__hash__', '__iadd__', '__imul__', '__init__', '__init_subclass__', '__iter__', '__le__', '__len__', '__lt__', '__mul__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__reversed__', '__rmul__', '__setattr__', '__setitem__', '__sizeof__', '__str__', '__subclasshook__', 'append', 'clear', 'copy', 'count', 'extend', 'index', 'insert', 'pop', 'remove', 'reverse', 'sort']
sum(x)
15
x
(1, 2, 3, 4, 5)

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
Traceback (most recent call last):
  File "C:/Users/HP/Desktop/46.py", line 20, in <module>
    print(sum(55,88))
TypeError: 'int' object is not iterable

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
(55, 88)
143

===================== RESTART: C:/Users/HP/Desktop/46.py ====================
(55, 3, 44)
102
(1, 2, 3, 4, 5, 6, 43, 12, 332.333)
408.333
sum(t)
Traceback (most recent call last):
  File "<pyshell#19>", line 1, in <module>
    sum(t)
NameError: name 't' is not defined
t=1,2,3,4,5,6,7,8,90,,66,77,88
SyntaxError: invalid syntax
t=(1,2,3,4,5,6,7,8,90,66,77,88)
sum(t)
357
min(t)
1
max(t)
90
mean(t)
Traceback (most recent call last):
  File "<pyshell#25>", line 1, in <module>
    mean(t)
NameError: name 'mean' is not defined
avereage(t)
Traceback (most recent call last):
  File "<pyshell#26>", line 1, in <module>
    avereage(t)
NameError: name 'avereage' is not defined
>>> t=("a",1,2,3,4,5,6,7,8,90,66,77,88)
>>> min(t)
Traceback (most recent call last):
  File "<pyshell#28>", line 1, in <module>
    min(t)
TypeError: '<' not supported between instances of 'int' and 'str'
>>> t=(str(a),1,2,3,4,5,6,7,8,90,66,77,88)
Traceback (most recent call last):
  File "<pyshell#29>", line 1, in <module>
    t=(str(a),1,2,3,4,5,6,7,8,90,66,77,88)
NameError: name 'a' is not defined
>>> a=12
>>> t=(str(a),1,2,3,4,5,6,7,8,90,66,77,88)
>>> min(t)
Traceback (most recent call last):
  File "<pyshell#32>", line 1, in <module>
    min(t)
TypeError: '<' not supported between instances of 'int' and 'str'
>>> t=(,1,2,3,4,5,6,7,8,90,66,77,88)
SyntaxError: invalid syntax
>>> t=(1,2,3,4,5,6,7,8,90,66,77,88)
>>> min(t)
1
>>> max(t)
90
>>> t.count(",")
0
>>> t.count("int")
0
>>> t.count("")
0
>>> t.len("")
Traceback (most recent call last):
  File "<pyshell#40>", line 1, in <module>
    t.len("")
AttributeError: 'tuple' object has no attribute 'len'
>>> len(t)
12
